test/commands/svc/enable.test.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, test } from '@oclif/test'
+import * as fs from 'node:fs'
+import * as path from 'node:path'
+import { TempDirHelper } from '../../helpers/temp-dir-helper'
+import { BundleDescriptorService } from '../../../src/services/bundle-descriptor-service'
+
+describe('svc enable', () => {
+  let bundleDirectory: string
+  let bundleDescriptorService: BundleDescriptorService
+  const tempDirHelper = new TempDirHelper(__filename)
+
+  before(() => {
+    bundleDirectory = tempDirHelper.createInitializedBundleDir('sample-bundle')
+  })
+
+  beforeEach(() => {
+    process.chdir(bundleDirectory)
+    bundleDescriptorService = new BundleDescriptorService(bundleDirectory)
+    const bundleDescriptor = bundleDescriptorService.getBundleDescriptor()
+    bundleDescriptorService.writeBundleDescriptor({
+      ...bundleDescriptor,
+      svc: []
+    })
+  })
+
+  after(() => {
+    fs.rmSync(path.resolve(tempDirHelper.tmpDir), {
```

Comment:
This is not needed, since it is already done by `TempDirHelper`

Reply:
done